microcosm_pubsub/codecs.py

```diff
@@ -87,10 +66,10 @@ class PubSubMessageCodec:
 
     """
 
-    def __init__(self, schema):
+    def __init__(self, schema: PubSubMessageSchema):
```

Comment:
By using typing here we're implicitly saying we're not supporting Python pre-3.6. Is that something we want?

Reply:
That was already done when we started using mm-3.0.

Reply:
Gabe is right, this ship has sailed. We're never going back.